tron/core/actionrun.py

```diff
@@ -759,7 +760,9 @@ def to_json(state_data: dict) -> Optional[str]:
                     "exit_status": state_data["exit_status"],
                     "attempts": [ActionRunAttempt.to_json(attempt) for attempt in state_data["attempts"]],
                     "retries_remaining": state_data["retries_remaining"],
-                    "retries_delay": state_data["retries_delay"],
+                    "retries_delay": state_data["retries_delay"].total_seconds()
+                    if state_data["retries_delay"]
+                    else None,
```

Comment:
```suggestion
                    "retries_delay": state_data["retries_delay"].total_seconds()
                    if state_data["retries_delay"] is not None
                    else None,
```

i think we'd want something like the above - unless there's no difference between None and 0 when deserializing back into a timedelta later :)

Reply:
Ah, true. I think it's ultimately the same in restart, but there is indeed a semantic difference.